cli/utils/utils.go

```diff
@@ -43,3 +43,20 @@ func ValidateOutputPathFileMode(fileMode os.FileMode) error {
 	}
 	return nil
 }
+
+// GetHomeDir gets the home directory. If we're in CI, then it's the CWD, otherwise assume it's
+// /home/seluser
+func GetHomeDir() (string, error) {
+	if (os.Getenv("SAUCE_ROOT_DIR") != "") {
+		return os.Getenv("SAUCE_ROOT_DIR"), nil
+	}
+	if (os.Getenv("SAUCE_VM") == "") {
+		return "/home/seluser", nil
+	}
+	workingDir, err := os.Getwd()
```

Comment:
@dpgraham what do you think about renaming this method to GetProjectDir()? Since its home dir on our containers, but we talked about moving that one more level to a project folder.

Reply:
> If we're in CI, then it's the CWD

I don't think that's true though. In CI, _our_ home directory is still /home/seluser. If it's the current working, the default in CI will be the repo name (or something else that the CI provider will dictate). Which is why we copy the files over to our designated home directory. Or did you find a way of how we can run tests outside of our project dir?

Reply:
@alexplischke I updated the comment to reflect what the function actually does.

If it's CI, it will set it to `/home/seluser`. If it's `SAUCE_VM`, and `SAUCE_ROOT_DIR` is set, it will set it to the working directory (where the process is run).